en/docs/install-and-setup/setup/api-controller/managing-rate-limiting-policies/throttle-policy-import-export.md

```diff
@@ -0,0 +1,133 @@
+# Importing and Exporting Rate Limiting Policies
```

Comment:
The topics and all should be consistent with other docs. For example, this document should be consistent with this one [1]. 
- The main topic should be, "Managing Rate Limiting Policies"

[1] https://apim.docs.wso2.com/en/4.2.0/install-and-setup/setup/api-controller/managing-common-api-policies/managing-common-api-policies/

Reply:
Fixed.